src/services/checkYourAnswersService.ts

```diff
@@ -89,7 +91,12 @@ export const getAnswers = (req: Request, acspData: AcspData, i18n: any): Answers
     let answers: Answers = {};
     answers.typeOfBusiness = typeOfBusinessTranslated(acspData.typeOfBusiness!, i18n);
     answers.roleType = roleTranslated(acspData.roleType!, i18n);
-    answers.workSector = sectorTranslated(acspData.workSector!, i18n);
+    if (acspData.workSector !== null) {
+        answers.workSector = sectorTranslated(acspData.workSector!, i18n);
```

Comment:
You have already checked if work Sector is null so I think you dont need to check it again like this
acspData.workSector!